models/model_test.js

```diff
@@ -32,4 +32,16 @@ const fetchArticleById = (article_id) => {
     })
 }
 
-module.exports = { fetchTopics, fetchArticles, fetchArticleById };
\ No newline at end of file
+const fetchCommentsByArticleId = (article_id) => {
+
+    if(isNaN(article_id) || article_id <= 0){
+        return Promise.reject({ status: 400, message: 'bad request'})
+    }
```

Comment:
^^ if the article_id isn't a number, surely PSQL will throw an error? Have a look at this part of the notes for how you would handle a PSQL error: https://notes.northcoders.com/courses/js-back-end/error-handling#psql-thrown-errors